praktikumsplaner-backend/src/test/java/de/muenchen/oss/praktikumsplaner/service/PraktikumsstellenServiceTest.java

```diff
@@ -549,4 +551,84 @@ public void testDeleteStudiumsPraktikumsstelleWithoutExisting() {
         });
 
     }
+
+    @Test
```

Comment:
please add tests for the scenario that requests are request for a stelle with an assigned nwk that are not allowed